src/main/java/at/hannibal2/skyhanni/features/garden/farming/FarmingWeightDisplay.kt

```diff
@@ -381,6 +394,7 @@ object FarmingWeightDisplay {
     private fun inGardenEnabled() = (LorenzUtils.inSkyBlock && GardenApi.inGarden()) || config.showOutsideGarden
 
     private fun isEtaEnabled() = config.overtakeETA
+    private fun isMonthlyLb() = config.eliteLbType.get() == EliteFarmingWeightConfig.EliteFarmingWeightLbType.MONTHLY
```

Comment:
make the last B upper case, as Leaderboard is a acronym and we write them uppercase when 2 chars long. for 3 char or longer we only uppercase the first char.